### PR TITLE
Update vagrantbox to version 0.7

### LINKED ISSUE #16: Update vagrant-box
## Feature description
Update [Vagrantfile](https://github.com/hannemariavister/terraform-nomad-nifi/blob/master/Vagrantfile.default#L3) with `config.vm.box_version = ">= 0.6, < 0.8"`.

## Checklist (after created issue)
- [x] Added label(s)
- [ ] Added to project
- [x] Added to milestone

### PR DESCRIPTION
**Pull request by hannemariavister**
## Closes/fixes/resolves issue(s)?
Closes #16 

## What was added/changed/fixed?
Update [Vagrantfile.default](https://github.com/hannemariavister/terraform-nomad-nifi/blob/master/Vagrantfile.default#L3), to `config.vm.box_version = ">= 0.6, < 0.8"`

## Checklist (after created PR)
- [x] Added reviewers
- [ ] Added assignee(s)
- [x] Added label(s)
- [ ] Added to project
- [x] Added to milestone

### CODE REVIEW

**Review by karimacherradi — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 ## [0.1.0 UNRELEASED]
 
 ### Added
+- Added 02_run_terraform.yml #6
 - Created a simple example for nifi #2
 - Added documentation for [example/standalone](example/standalone) in README.md #8
+
 ### Changed
+- Update vagrant-box, `version = ">= 0.6, < 0.8"` #16
```
Comment (zhenik) [inline CHANGELOG.md]: I would recommend to stay in one major version. 

```suggestion
- Update vagrant-box, `version = ">= 0.7, < 0.8"` #16
```
ps.  0.8.0 is not released yet. Once it released you can go for `config.vm.box_version = ">= 0.8, < 0.9"`

File: Vagrantfile.default [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Vagrant.configure("2") do |config|
     config.vm.box = "fredrikhgrelland/hashistack"
-    config.vm.box_version = ">= 0.5, < 0.6"
+    config.vm.box_version = ">= 0.6, < 0.8"
```
Comment (zhenik) [inline Vagrantfile.default]:
```suggestion
    config.vm.box_version = ">= 0.7, < 0.8"
```

**Review by zhenik — CHANGES_REQUESTED** [review]

File: Vagrantfile.default (hunk 2 above)
Comment (hannemariavister) [reply Vagrantfile.default]: Thanks 😊

**Pull request merged**